gatsby-config.js

```diff
@@ -37,6 +37,8 @@ module.exports = {
         icon: `src/images/gatsby-icon.png`, // This path is relative to the root of the site.
       },
     },
+    `gatsby-plugin-typescript`,
+    `gatsby-plugin-tslint`,
```

Comment:
TSLint isn't recommended in TypesCript projects as it's no longer developed. It could be better to use ESLint instead